src/components/Modal/ModalWrapper/index.tsx

```diff
@@ -5,18 +5,28 @@ import ModalContext from '../../../context/modal-context';
 import { ModalWrap, Overlay, ButtonWrap } from './style';
 
 interface Props {
-	open: boolean;
 	children: any;
+	wrapperId: string;
+	closeCondition: boolean;
 	onClose: any;
 }
 
-const ModalWrapper = ({ open, children, onClose }: Props) => {
+const ModalWrapper = ({children, wrapperId, closeCondition, onClose}: Props) => {
 	const modalCtx = useContext(ModalContext);
+	let elementAppend = document.getElementById(wrapperId);
```

Comment:
this logic is not needed here, we should be able to accept any element object or element ref.